src/Fable.Arch/Fable.Arch.Virtualdom.Html.fs

```diff
@@ -0,0 +1,369 @@
+    | TextEmphasisStyle of obj
+    | TextHeight of obj
+    | TextIndent of obj
+    | TextJustifyTrim of obj
+    | TextKashidaSpace of obj
+    | TextLineThrough of obj
+    | TextLineThroughColor of obj
+    | TextLineThroughMode of obj
+    | TextLineThroughStyle of obj
+    | TextLineThroughWidth of obj
+    | TextOverflow of obj
+    | TextOverline of obj
+    | TextOverlineColor of obj
+    | TextOverlineMode of obj
+    | TextOverlineStyle of obj
+    | TextOverlineWidth of obj
+    | TextRendering of obj
+    | TextScript of obj
+    | TextShadow of obj
+    | TextTransform of obj
+    | TextUnderlinePosition of obj
+    | TextUnderlineStyle of obj
+    | Top of obj
+    | TouchAction of obj
+    | Transform of obj
+    | TransformOrigin of obj
+    | TransformOriginZ of obj
+    | TransformStyle of obj
+    | Transition of obj
+    | TransitionDelay of obj
+    | TransitionDuration of obj
+    | TransitionProperty of obj
+    | TransitionTimingFunction of obj
+    | UnicodeBidi of obj
+    | UnicodeRange of obj
+    | UserFocus of obj
+    | UserInput of obj
+    | VerticalAlign of obj
+    | Visibility of obj
+    | VoiceBalance of obj
+    | VoiceDuration of obj
+    | VoiceFamily of obj
+    | VoicePitch of obj
+    | VoiceRange of obj
+    | VoiceRate of obj
+    | VoiceStress of obj
+    | VoiceVolume of obj
+    | WhiteSpace of obj
+    | WhiteSpaceTreatment of obj
+    | Width of obj
+    | WordBreak of obj
+    | WordSpacing of obj
+    | WordWrap of obj
+    | WrapFlow of obj
+    | WrapMargin of obj
+    | WrapOption of obj
+    | WritingMode of obj
+    | [<Erase>] VAttr of string * Value
+    
+[<KeyValueList>]
+type VProps<'T> = 
+    | [<CompiledName("attributes")>] Attrs of VAttr list
+    | Key of string
+    | Alt of string
+    | Src of string
+    | Href of string
+    | Width of Value
+    | Height of Value
+    | Value of string
+    | Placeholder of string
+    | Disabled of bool
+    | EncType of string
+    | Form of string
+    | FormAction of string
+    | FormEncType of string
+    | FormMethod of string
+    | FormNoValidate of bool
+    | FormTarget of string
+    | Default of bool
+    | Defer of bool
+    | Cols of int
+    | ColSpan of int
+    | Checked of bool
+    | CharSet of string
+    | CellPadding of int
+    | CellSpacing of int
+    | Async of bool
+    | AutoComplete of string
+    | AutoFocus of bool
+    | AutoPlay of bool
+    | Capture of bool
+    | DefaultChecked of bool
+    | DefaultValue of U2<string, ResizeArray<string>>
+    | Accept of string
+    | AcceptCharset of string
+    | AccessKey of string
+    | Action of string
+    | AllowFullScreen of bool
+    | AllowTransparency of bool
+    | [<Erase>] VProp of string * Value
+    | [<CompiledName("ev-click")>] OnClick of (obj -> 'T)
+
+
+type VTag = string
+
+
+
+type VNode<'T> = 
+    | Plain of string
+    | Unary of VTag * VProps<'T> list
+    | Nested of VTag * VProps<'T> list * VNode<'T> list
+
+
+/// Example usage
```

Comment:
Ok no problem. 
If you could add an independant sample that would be great then. Just to help see how everything is fitting :)